src/main/g8/build.sbt

```diff
@@ -92,8 +93,7 @@ sparkEmrConfigs := List(
     "spark.shuffle.service.enabled" -> "true",
     "spark.rdd.compress" -> "true",
     "spark.driver.extraJavaOptions" -> "-Djava.library.path=/usr/local/lib",
-    "spark.executor.extraJavaOptions" -> "-Djava.library.path=/usr/local/lib -XX:+UseParallelGC",
-    "spark.executorEnv.LD_LIBRARY_PATH" -> "/usr/local/lib"
+    "spark.executor.extraJavaOptions" -> "-XX:+UseParallelGC"
   ),
   EmrConfig("spark-env").withProperties(
     "LD_LIBRARY_PATH" -> "/usr/local/lib"
```

Comment:
It's enough only to set `LD_LIBRARY_PATH` in a `spark-env` setting to make GDAL and PDAL work.